backend/development.ini.sample

```diff
@@ -770,6 +770,13 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 
 ; translation_service.enabled = False
 
+
+## Translation service timeout in seconds.
+# This timeout will be applied to each translation request.
+; translation_service.timeout = 0
+# example:
+# translation_service.timeout = 5
```

Comment:
Maybe add bigger timeout value like 10-20s

I think 5sec is to slow for a timeout on this kind of feature.

Reply:
Done.